chain-signatures/node/src/rpc_client.rs

```diff
@@ -12,6 +12,7 @@ pub async fn fetch_mpc_contract_state(
 ) -> anyhow::Result<ProtocolState> {
     let protocol_state: mpc_contract::ProtocolContractState =
         rpc_client.view(mpc_contract_id, "state").await?.json()?;
+    tracing::debug!(?protocol_state, "fetched protocol state");
```

Comment:
we already have a debug for contract state

Reply:
+1